website/client/src/components/challenges/findChallenges.vue

```diff
@@ -243,6 +243,11 @@ export default {
       this.page += 1;
       this.loadChallenges();
     }, 1000),
+    userCanViewFlagged (challenge) {
+      return challenge.flagCount < 2
+        || this.user.contributor.admin
+        || this.user._id === challenge.leader._id;
```

Comment:
Is this meant to allow the creator of the Challenge to see it in their list even if it's been flagged and hidden? If that's intended, it's not working because the Challenge is getting filtered out during the database query when the user does a GET on their list.

Reply:
Great point. Actually, if the flag-hiding is all done via the API, we should assume that any challenges returned by the API are challenges the user can see, meaning this filtering is unnecessary. I'll update the API code to ensure the challenge owner can see flagged challenges.